docs/source/analyze/image-pull-secrets.md

```diff
@@ -3,23 +3,28 @@ title: Image Pull Secrets
 description: Analyzing the image pull secrets that are available in the cluster
 ---
 
+The `imagePullSecrets` analyzer uses data from the [clusterResources collector](https://troubleshoot.sh/collect/cluster-resources).
+The `clusterResources` collector is automatically added and will always be present.
+
+## Parameters
+
+**registryName**: (Required) The name of the registry to check
+
 ```yaml
 apiVersion: troubleshoot.sh/v1beta2
 kind: Preflight
 metadata:
   name: preflight-sample
 spec:
   analyzers:
-    - secret:
-        checkName: PG URI
-        secretName: postgres
-        namespace: default
-        key: uri
+    - imagePullSecret:
+        checkName: Pull from Quay
+        registryName: quay.io
         outcomes:
           - fail:
-              message: You don't have a pg uri secret
+              message: You won't be able to pull from Quay
           - pass:
-              message: Probably a green light connecting to pg
+              message: Images can be pulled from Quay
```

Comment:
Does this actually test the ability to pull from quay.io, or just test that there's a secret for that registry? The secret could be invalid.

Reply:
Only that a secret exists for that registry.